### PR TITLE
bugfix/1878_gap-below-component

### LINKED ISSUE #1878: input-file & button-link: gap below component
There is always a little gap below an input-file/button-link.

Screenshot 2020-07-16 at 09 54 48

### PR DESCRIPTION
**Pull request by raphael-suter**
Fixes #1878 

# Done is when (DoD):
- [x] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [x] The modified component properties have been added to the typescript typings.
- [x] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/components/10-atoms/input-file/index.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+axa-input-file {
```
Comment (danyball) [inline src/components/10-atoms/input-file/index.scss:1]: This selector could be invalid if someone uses the new component versioning like `axa-input-file-pod_xy`. Better set display to the internal elements or create a wrapper div for that.

File: src/components/20-molecules/datepicker/index.js [hunk 2]
```diff
@@ -1,28 +1,27 @@
 /* eslint-disable import/no-extraneous-dependencies */
+import AXADropdown from '@axa-ch/dropdown';
```
Comment (danyball) [inline src/components/20-molecules/datepicker/index.js:2]: Something changed at that file? many changes, but just reordering?

**Review by danyball — CHANGES_REQUESTED** [review]

File: src/components/20-molecules/datepicker/index.js (hunk 2 above)
Comment (raphael-suter) [reply src/components/20-molecules/datepicker/index.js:2]: No, there aren't any changes here, except the reordered imports (that was my commit to re-run the checks).

File: src/components/10-atoms/input-file/index.scss (hunk 1 above)
Comment (raphael-suter) [reply src/components/10-atoms/input-file/index.scss:1]: As discussed before, this shouldn't be a problem.

File: src/components/10-atoms/input-file/index.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+  display: inline-block;
```
Comment (danyball) [inline src/components/10-atoms/input-file/index.scss:2]: Why do you add display? At my chrome everything works without it.

Comment (raphael-suter) [reply src/components/10-atoms/input-file/index.scss:2]: Without it the component doesn't get selected properly & we could use it to allow 100% width. (Would have to change another display inside the component)

Screenshot 2020-07-21 at 08 49 07

Comment (danyball) [reply src/components/10-atoms/input-file/index.scss:2]: Very nice!!! Please add this new functionality to the readme.

**Review by danyball — CHANGES_REQUESTED** [review]

**Review by danyball — DISMISSED** [review]

**Review by danyball — APPROVED** [review]

**Pull request merged**